### PR TITLE
Created a Product card UI Design

### PR DESCRIPTION
**Pull request by Abbas4tech**

### CODE REVIEW

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
+* {
+  margin: 0%;
+  padding: 0%;
+  font-family: "Poppins", sans-serif;
+}
+body {
```
Comment (mmusk) [inline tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:6]: wrap this in a different div its not a good practise to give these properties to body 
                                                                                                              ~Mezab OP

Comment (coder-mezab) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:6]: @mmusk faran bhai told  me this so its said by @faran4engg 😁

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

**Review by nehalskdev — APPROVED** [review]

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Product Card UI Design</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```
Comment (shaikh-7abish) [inline tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html:9]: Always link `CSS` file in last.

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  transition: 0.5s;
+  opacity: 0;
+  visibility: hidden;
+}
+.main .card:hover .contentBx .size {
+  opacity: 1;
+  visibility: visible;
+  transition-delay: 0.6s;
+}
+.main .card:hover .contentBx .color {
```
Comment (shaikh-7abish) [inline tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:101]: Use multiple selector here `,`

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  border-radius: 4px;
+  text-align: center;
+  line-height: 26px;
+  font-size: 14px;
+  display: inline-block;
+  color: #111;
+  background: #fff;
+  margin: 0 5px;
+  transition: 0.5s;
+}
```
Comment (shaikh-7abish) [inline tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:127]: group these properties.

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+  padding: 10px 20px;
+  border-radius: 4px;
+  margin-top: 10px;
+  text-decoration: none;
+  font-weight: 600;
+  color: #111;
+  opacity: 0;
+  transform: translate(50px);
+  transition: 0.5s;
+}
```
Comment (shaikh-7abish) [inline tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:160]: This one too.

**Review by nehalskdev — APPROVED** [review]

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html (hunk 2 above)
Comment (faran4engg) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html:9]: No, Why would you do that. ?

Comment (shaikh-7abish) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html:9]: Sorry, I've been mistaken.

Comment (shaikh-7abish) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html:9]: I taught loading CSS first will load it in default font, and then apply the custom font,

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+            <span>9</span>
+            <span>10</span>
+          </div>
+          <div class="color">
+            <h3>Color :</h3>
+            <span></span>
+            <span></span>
+            <span></span>
+          </div>
+          <a href="">Buy Now!</a>
```
Comment (shaikh-7abish) [inline tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html:35]: Use `#` in href att.

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html (hunk 2 above)
Comment (shaikh-7abish) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html:9]: I taught loading CSS first will load it in default font, and then apply the custom font.

**Review by nasir-k — APPROVED** [review]

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css (hunk 5 above)
Comment (Abbas4tech) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:160]: Done.

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css (hunk 4 above)
Comment (Abbas4tech) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:127]: Done.

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css (hunk 1 above)
Comment (Abbas4tech) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:6]: Yes Done. @mmusk

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html (hunk 2 above)
Comment (Abbas4tech) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html:9]: Yes Done. @shaikh-7abish

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html (hunk 6 above)
Comment (Abbas4tech) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/index.html:35]: Yes Done.

File: tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css (hunk 3 above)
Comment (Abbas4tech) [reply tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css:101]: I didn't intended to select all those classes, i used descended selector there. why would i use multiple selector when i dont want to select those classes as well?

**Pull request merged**